resource_management/azure_mgmt_compute/spec/usage_operations_spec.rb

```diff
@@ -8,8 +8,9 @@
 include Azure::ARM::Compute
 
 describe UsageOperations do
-  before(:all) do
-    @client = COMPUTE_CLIENT.usage_operations
+  before(:each) do
+    @resource_helper = ResourceHelper.new
```

Comment:
should we be instantiating everything that's in ResourceHelper every time? Don't we need only compute client in this case?